include/swift/AST/SourceFile.h

```diff
@@ -63,11 +67,17 @@ class SourceFile final : public FileUnit {
   struct ImportedModuleDesc {
     ModuleDecl::ImportedModule module;
     ImportOptions importOptions;
+
+    // Filename for a @_private import.
     StringRef filename;
 
+    // Names of explicitly imported SPIs.
+    SmallVector<Identifier, 4> spis;
```

Comment:
Should this be an arena-allocated ArrayRef, just like the ImportedModuleDesc itself?

Reply:
Right, it should now be fixed.